fe/fe-core/src/main/java/com/starrocks/scheduler/Constants.java

```diff
@@ -34,4 +34,6 @@ public enum TaskRunState {
         SUCCESS,
     }
 
+    public static final int DEFAULT_TASK_RUN_PRIORITY = 0;
+    public static final int MANUAL_TASK_RUN_PRIORITY = 20;
```

Comment:
prefer using a enum, not respective constant value

Reply:
priory should be integer and in java enum is not easy to convert to each other